src/test/custom_nodes/node_add_sub.c

```diff
@@ -77,7 +77,7 @@ int execute(const struct CustomNodeTensor* inputs, int inputsCount, struct Custo
     return 0;
 }
 
-// TODO: Some unit tests are based on a fact that this node library is dynamic and can take shape{1,3} as input.
+// Some unit tests are based on a fact that this node library is dynamic and can take shape{1,3} as input.
 // This needs to be addressed before release.
```

Comment:
"This needs to be addressed before release." - do we have it in scope?

Reply:
I don't think this is required for release as it affects only unit tests. But I changed the expected metadata & expected values in tests in case further on we decide to handle treating "0" as elastic shape.